src/main/java/vo/Lotto.java

```diff
@@ -0,0 +1,42 @@
+package vo;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import util.LottoValidator;
+
+public class Lotto {
+
+    private final ArrayList<LottoNumber> numbers;
```

Comment:
<img width="1024" height="588" alt="Image" src="https://github.com/user-attachments/assets/fc9446ac-5e29-4ff8-8a92-924cc84f3c02" />
다형성을 위해서는 ArrayList 말고 List 인터페이스로 선언해주는게 좋아요.

```suggestion
    private final List<LottoNumber> numbers;
```

- 관련 글 중 하나: https://velog.io/@dnwlsrla40/List-ArrayList-배열-선언-시-List와-ArrayList-차이